AppAuth-JS/built/flags.d.ts

```diff
@@ -0,0 +1,2 @@
+export declare const IS_LOG: boolean;
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork